tests/static/js/formatters-internal/image.js

```diff
@@ -1,106 +1,197 @@
 import Formatters from 'static/js/formatters.js';
 
 describe('image formatter', () => {
-  const usUrl = 'https://a.mktgcdn.com/p/1024x768.jpg';
-  const euUrl = 'https://dyn.eu.mktgcdn.com/f/0/FOO.jpg';
-  const usImg = {url: usUrl};
-  const euImg = {
-    url: euUrl,
-    width: 1024,
-    height: 768,
-    sourceUrl: 'https://a.mktgcdn.com/p/FOO/1024x768.jpg',
+  const photoUrl = 'https://dynm.mktgcdn.com/p/FOO/2000x1000.jpg/';
+  const photoWithPaddingUrl = 'https://dynm.mktgcdn.com/p/FOO/1.0000/2000x1000.jpg/';
+  const oldFileUrl = 'https://a.mktgcdn.com/f/0/FOO.jpg';
```

Comment:
I thought only EU URLs were in the file format with the `/f/`, not US?

Reply:
It doesn't hurt to have extra test coverage, going to keep. Ack

Reply:
I think we're missing a case though. The current euFileUrl is actually the old format, and we also need to test for the new EU file url format without the account id, right?

Reply:
I don't think it's worth/possible to add a test case for every single possible case. The "with account id" case is already covered by newFileUrl. euFileUrl is mainly to test for the eu domain. Ack

Reply:
I think that's the opposite. Based on [this](https://yext.slack.com/archives/C0DA5HJCT/p1713445795909829) thread, the new behavior is **without** the account id, specifically for EU file URLs, which is why it's important to test this case. Going forward, the two types of URLs we'll get is US photo URLs and EU file URLs without account id, so I think we should cover those cases, even if we don't cover others like file URLs with account id in US domain

Reply:
done